consumer/src/main/kotlin/streams/StreamsEventSinkExtensionFactory.kt

```diff
@@ -51,27 +51,21 @@ class StreamsEventSinkExtensionFactory : KernelExtensionFactory<StreamsEventSink
                                 logService.getUserLog(StreamsEventSinkQueryExecution::class.java),
                                 strategyMap)
 
-                        // Create and start the Sink
+                        // Create the Sink
+                        val log = logService.getUserLog(StreamsEventSinkFactory::class.java)
                         eventSink = StreamsEventSinkFactory
                                 .getStreamsEventSink(configuration,
                                         streamsQueryExecution,
                                         streamsTopicService,
-                                        logService.getUserLog(StreamsEventSinkFactory::class.java))
-                        eventSink.start()
-                        if (Neo4jUtils.isWriteableInstance(db)) {
-                            if (streamsLog.isDebugEnabled) {
-                                streamsLog.debug("Subscribed topics with Cypher queries: ${streamsTopicService.getAllCypherTemplates()}")
-                                streamsLog.debug("Subscribed topics with CDC configuration: ${streamsTopicService.getAllCDCTopics()}")
-                            } else {
-                                streamsLog.info("Subscribed topics: ${streamsTopicService.getTopics()}")
-                            }
-                        }
+                                        log,
+                                        db)
+                        // start the Sink
+                        Neo4jUtils.executeInLeader(db, log) { initSinkModule() }
 
                         // Register required services for the Procedures
                         StreamsSinkProcedures.registerStreamsSinkConfiguration(streamsSinkConfiguration)
                         StreamsSinkProcedures.registerStreamsEventConsumerFactory(eventSink.getEventConsumerFactory())
                         StreamsSinkProcedures.registerStreamsEventSinkConfigMapper(eventSink.getEventSinkConfigMapper())
-                        streamsLog.info("Streams Sink module initialised")
```

Comment:
why did you remove this?

Reply:
I moved it into the `initSinkModule` method